### PR TITLE
Feature login view

### PR DESCRIPTION
**Pull request by ak2022dev**
Please review login  page created to be linked to home page. Currently visual but no function when submit pressed.

### CODE REVIEW

File: spec/features/view_login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  context "basic form test" do
+    it "Displays title message, username and password input fields and button to submit" do
+      visit('/makersbnb/login')
+      expect(page).to have_content "Login page"
+      fill_in 'username', with: 'myusername'
+      fill_in 'password', with: 'mypassword'
+      click_button('Submit')
+    end
+  end
+end
```
Comment (kaykakaraka) [inline spec/features/view_login_spec.rb:18]: great!

File: views/makersbnb/login.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+<form>
+  <label>Username</label>  
+  <input type="text" name="username" /><br>
+  <label>Password</label>
+  <input type="password" name="password" /><br>
+  <input type="submit" name="Submit" />
+</form>
+
```
Comment (kaykakaraka) [inline views/makersbnb/login.erb:15]: really clear, looks good

**Review by kaykakaraka — APPROVED** [review]
Great!

**Review by BenjaminNeustadt — APPROVED** [review]

**Comment by BenjaminNeustadt** [comment]
Looks great work! Thanks Amir

**Pull request merged**